src/main/java/com/quirko/gui/Main.java

```diff
@@ -14,21 +16,24 @@
 
     @Override
     public void start(Stage primaryStage) throws Exception {
-
         URL location = getClass().getClassLoader().getResource("gameLayout.fxml");
         ResourceBundle resources = null;
         FXMLLoader fxmlLoader = new FXMLLoader(location, resources);
         Parent root = fxmlLoader.load();
         GuiController c = fxmlLoader.getController();
-
+        DifficultySetPanel dfs= new DifficultySetPanel();
+        Enum difficultyLevel = dfs.getDifficultyLevel();
+        while(difficultyLevel == null){
+            difficultyLevel = dfs.getDifficultyLevel();

```

Comment:
Hmm, neden böyle bir loopta bekliyoruz?

Reply:
Hocam burada başlangıçta null olarak tanımlı olduğu için difficultyLevel butona basıp değer algılanınca çalışması açısından böyle yazdım. Yukarıda da sleeple bu yüzden saniye bekletiyorum.